notebooks/Tutorial_Top-K_Multidimensional_Motif_and_Matches_Discovery.ipynb

```diff
@@ -0,0 +1,827 @@
+{
```

Comment:
<p>I think this should be:</p><pre class="ql-syntax" spellcheck="false">for colname in df.columns:
&nbsp; &nbsp; plt.plot(df[colname], label=colname)
plt.legend()
plt.show()
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/557/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>